hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OmTableInsightTask.java

```diff
@@ -489,7 +489,7 @@ private HashMap<String, Long> initializeSizeMap(boolean replicated) {
   }
 
   public static String getTableCountKeyFromTable(String tableName) {
-    return tableName + "TableCount";
+    return tableName + "Count";
   }
 
   public static String getReplicatedSizeKeyFromTable(String tableName) {
```

Comment:
We should add test case for `ReplicatedDataSize` and `UnReplicatedDataSize` as well so that it will not break in future.

Reply:
Done thanks!